pm_asephonons/pm_asephonons.xml

```diff
@@ -1,4 +1,5 @@
-<tool id="pm_asephonons" name="Generate phonons using ASE and DFTB+ (pm-asephonons)" version="0.1.0" python_template_version="3.5" profile="22.01">
+<tool id="pm_asephonons" name="pm-asephonons" version="0.1.0" python_template_version="3.5" profile="22.01">
+    <description>generate phonons using ASE and DFTB+</description>
```

Comment:
In a physical sense, phonons are intrinsic to a material, so we don't _generate_ them per se (pedantic, but the phrasing feels very off to me)

```suggestion
    <description>calculate phonons using ASE and DFTB+</description>
```